packaging/ap/ubuntu/generic/config

```diff
@@ -8,55 +8,80 @@ get_prop() {
 
 if [ "$1" = "configure" ] || [ "$1" = "reconfigure" ]; then
 
-  db_input high harmony-ap/usedynamicdiscovery
-  db_go
-
-  db_get harmony-ap/usedynamicdiscovery
-  USEDYNAMIC="$RET"
+  if [[ -z "${USE_DYNAMIC}" ]]; then
+    db_input high harmony-ap/usedynamicdiscovery
+    db_go
+  else 
+    db_set harmony-ap/usedynamicdiscovery "$USE_DYNAMIC"
+    USEDYNAMIC=$USE_DYNAMIC
+  fi
 
   if [ "$USEDYNAMIC" = true ]; then
-    db_input high harmony-ap/smlzone
+    if [[ -z "${SML_ZONE}" ]]; then
+      db_input high harmony-ap/smlzone
+      db_go
+    else 
+      db_set harmony-ap/smlzone "$SML_ZONE"
+    fi
   fi
 
   db_beginblock
-  db_input high harmony-ap/adminuser || true
-  db_input high harmony-ap/adminpassword || true
-  db_input high harmony-ap/partyname || true
-  db_input high harmony-ap/serverdn || true
-  db_input high harmony-ap/tomcatport || true
+  if [[ -z "${ADM_USER}" ]]; then
+    db_input high harmony-ap/adminuser || true
+  else 
+    db_set harmony-ap/adminuser "$ADM_USER"
+  fi
+  if [[ -z "${ADM_PASS}" ]]; then
+    db_input high harmony-ap/adminpassword || true
+  else 
+    db_set harmony-ap/adminpassword "$ADM_PASS"
+  fi
+  if [[ -z "${PARTY_NAME}" ]]; then
+    db_input high harmony-ap/partyname || true
+  else 
+    db_set harmony-ap/partyname "$PARTY_NAME"
+  fi
+  if [[ -z "${SERVER_DN}" ]]; then
+    db_input high harmony-ap/serverdn || true
+  else 
+    db_set harmony-ap/serverdn "$SERVER_DN"
+  fi
+  if [[ -z "${TOMCAT_PORT}" ]]; then
+    db_input high harmony-ap/tomcatport || true
+  else 
+    db_set harmony-ap/tomcatport "$TOMCAT_PORT"
+  fi
   db_endblock
   db_go
 
   db_beginblock
-  DB_HOST="$(get_prop domibus.database.serverName)"
-  if [[ -n "$DB_HOST" ]]; then
+  if [[ -z "${DB_HOST}" ]]; then
+    db_input high harmony-ap/database-host
+  else 
     db_set harmony-ap/database-host "$DB_HOST"
   fi
-  db_input high harmony-ap/database-host
-  DB_PORT="$(get_prop domibus.database.port)"
```

Comment:
This is a regression. It is important that the configuration file can be used to override the configuration database value.